src/lib.rs

```diff
@@ -175,17 +175,47 @@ impl<'text> InitialInfo<'text> {
         text: &'a str,
         default_para_level: Option<Level>,
     ) -> InitialInfo<'a> {
+        let ext_info = InitialInfoExt::new_with_data_source(data_source, text, default_para_level);
+        ext_info.base
+    }
+}
+
+/// Extended version of InitialInfo (not public API).
+#[derive(PartialEq, Debug)]
+struct InitialInfoExt<'text> {
+    base: InitialInfo<'text>,
+    pure_ltr: Vec<bool>,
```

Comment:
nit: please document this field

Reply:
hm, doesn't seem to have changed?

Reply:
Errr.... I must have goofed, lemme try again....!